docker/star/Dockerfile

```diff
@@ -9,22 +9,25 @@ RUN mkdir build
 WORKDIR build 
 # install additional python packages
 #Install wget, unzip
-RUN apt update && apt install -y liblzma-dev libbz2-dev cmake automake curl libboost-all-dev wget build-essential gcc-multilib zlib1g-dev libxml2-dev libncurses5-dev
+RUN apt update && apt install -y liblzma-dev \
+	libbz2-dev \
+	cmake automake \
+	curl \
+	libboost-all-dev \
+	wget \
+	build-essential \
+	gcc-multilib \
+	zlib1g-dev \
+	libxml2-dev \
+	libncurses5-dev \
+	r-base \
+	r-base-core \
+	r-base-dev 
 
 RUN wget https://github.com/alexdobin/STAR/archive/2.5.3a.tar.gz && \
```

Comment:
Could you explain why we have a STAR and a STAR+RSEM docker?

Reply:
should say "RSEM+STAR". RSEM alone has the ability to use aligners(bowtie, bowtie2, star..) to do alignment.